tests/sequences/func_test.py

```diff
@@ -1,5 +1,5 @@
-import pytest
-
+import pytest 
+from sequences.func import remove_duplicate_lists
```

Comment:
It's better to include new function to the package `__init__` module.
This will make it available via the package import at line 3.

Follow the general concept in https://github.com/edu-python-course/problem-sets/blob/4e056d7bb7e3f1645b9c72e1343a40acf766ca52/src/sequences/__init__.py#L34C1-L56C1.